core/appcontainer/MessageSourceModel.js

```diff
@@ -39,6 +39,11 @@ export class MessageSourceModel {
         return this.message(config);
     }
 
+    prompt(config) {
+        config = defaults({}, config, {confirmText: 'OK', cancelText: 'Cancel', input: true});
+        return this.message(config);
```

Comment:
here is where we can install the default textInput

Reply:
Unfortunately, since MessageSourceModel is cross-platform and the TextInputs are platform specific, the only way to do the defaulting here would be to use dynamics.

Instead, I think its simpler to default in the platform-specific Message components. However, I changed it so that all the aspects of the input spec are optional. Passing an empty input spec tells the message to render a form with all the defaults (i.e. a TextInput, no default value and no validation rules).